src/components/Lobby/GameView/GameView.tsx

```diff
@@ -12,17 +11,40 @@ import dayjs from 'dayjs';
 import { faArrowRight } from '@fortawesome/free-solid-svg-icons';
 
 import { useTranslation } from 'react-i18next';
+import Modal from '../../Common/Modal/Modal';
+import DrawingSelector from './DrawingSelector/DrawingSelector';
+import { Col, Row } from 'react-grid-system';
+import { YesOrNo } from '../../../../server/classes/Votes/YesNoVote';
+import { Socket } from 'socket.io-client';
 
 interface GameProps {
 	game: Game;
+	updateLobby: (lobby: Lobby) => void;
+	socket: typeof Socket;
 }
 
 export default function GameView(props: GameProps): JSX.Element {
-	const socket = useSocketLobby();
 	const { t } = useTranslation();
 	const [playerId] = useLocalStorage(LocalStorageKey.PlayerId);
 
 	const [currentPlayer, setCurrentPlayer] = useState<Player>(props.game.players[props.game.currentPlayerIndex])
+	const [isStartVoteModalVisible, setIsStartVoteModalVisible] = useState(false);
+	const [selectedDrawing, setSelectedDrawing] = useState<number | undefined>(1);
+	const [isCurrentVoteModalVisible, setIsCurrentVoteModalVisible] = useState(false);
+
+	useEffect(() => {
+		props.socket.on('start-vote', (lobby: Lobby) => {
+			setIsCurrentVoteModalVisible(true)
+			props.updateLobby(lobby);
+		})
+
+		props.socket.on('stop-vote', (lobby: Lobby) => {
+			setIsCurrentVoteModalVisible(false)
+			props.updateLobby(lobby);
+		})
+
+		// eslint-disable-next-line react-hooks/exhaustive-deps
+	}, [props.socket])
```

Comment:
Tu ajoutes des évènements mais tu ne les enlèves jamais ? Est-ce normal ?
Que ce passe t-il si tu ajoutes deux évènements avec le même nom ?

Reply:
Nope, I completely forgot to remove them, good call!

Here's how two events with the same name are handled: you can actually add multiple callbacks to a same event name. So, it would simply trigger both callbacks.

I will remove the listener in the cleanup function.

Reply:
To add more to this point, you can then remove a single listener (so, if we have two listeners for start-vote, we can remove 1 listener and still have 1 listener trigger), or remove all of them at the same time. I will remove all of them, just in case.